src/Components/Web/src/ElementReferenceExtensions.cs

```diff
@@ -17,7 +17,19 @@ public static class ElementReferenceExtensions
         /// </summary>
         /// <param name="elementReference">A reference to the element to focus.</param>
         /// <returns>The <see cref="ValueTask"/> representing the asynchronous focus operation.</returns>
-        public static ValueTask FocusAsync(this ElementReference elementReference)
+        public static ValueTask FocusAsync(this ElementReference elementReference) => elementReference.FocusAsync();
```

Comment:
This likely needs to be: 

```suggestion
        public static ValueTask FocusAsync(this ElementReference elementReference) => elementReference.FocusAsync(preventScroll: false);
```

Reply:
yes

Reply:
Yes. Fix it